tests/functional/api/test_merge_requests.py

```diff
@@ -27,9 +27,11 @@ def test_merge_requests(project):
             "commit_message": "New commit in new branch",
         }
     )
-    project.mergerequests.create(
+
+    mr = project.mergerequests.create(
         {"source_branch": "branch1", "target_branch": "main", "title": "MR readme2"}
     )
+    assert mr.id is not None
```

Comment:
What's this?

Reply:
Wow thanks for that, brainfart during rebase 😁 fixed.